netbox/dcim/tables/devices.py

```diff
@@ -285,14 +285,14 @@ class DeviceTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable):
     class Meta(NetBoxTable.Meta):
         model = models.Device
         fields = (
-            'pk', 'id', 'name', 'status', 'tenant', 'tenant_group', 'device_role', 'manufacturer', 'device_type',
+            'pk', 'id', 'name', 'status', 'tenant', 'tenant_group', 'role', 'manufacturer', 'device_type',
```

Comment:
Renaming the column might cause an issue with users' saved table preferences. We may be able to address this in the database migration, updating any occurrence of `device_role` in `tables.DeviceTable.columns`.

Reply:
I've added a routine in the migration file to update references to the old `device_role` table column in user configs.